File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.py
class Solution:
    def arrayStringsAreEqual(self, word1: List[str], word2: List[str]) -> bool:
        arr1  = ""
        arr2  = ""
        for i in word1:
            arr1 += i
        for i in word2:
            arr2 += i
        if arr1 == arr2:
            return True
        return False

File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.java
class Solution {
    public boolean arrayStringsAreEqual(String[] word1, String[] word2) {
        String arr1 = "";
        String arr2 = "";
        for (int i = 0; i < word1.length;i++){
            arr1 += word1[i];
        }
        for (int i = 0; i < word2.length;i++){
            arr2 += word2[i];
        }
        System.out.println(arr1);
        System.out.println(arr2);
        return arr1.equals(arr2) ? true : false;   
    }
}